core/sqf/sql/scripts/sqcheck

```diff
@@ -33,7 +33,7 @@ function Usage {
     echo 
     echo "Usage: $0 [ -c <cc> | -i <nn> | -d <nn> | -h | -f | -q | -v | -r | -j ]"
     echo 
-    echo "-i <nn>  Number of times the check for SQ processes be done (Default $max_checks)"
+    echo "-i <nn>  Number of times the check for Trafodion processes be done (Default $max_checks)"
```

Comment:
Do we want to say, "Number of times the check for Trafodion processes **will** be done (Default $max_checks)"